README.md

```diff
@@ -149,8 +131,25 @@ $ docker run -it --volume "/usr/share/fonts:/fonts" --volume "$(pwd):/data" --us
 
 #### Nix
 
-The `nix` package manager is available as a standalone package manager on many platforms and is a viable way to run SILE.
-See usage in the [NixOS section](#nixos).
+The `nix` package manager is available as a standalone package manager on many platforms other than NixOS; like most Linux and BSD distributions, macOS, and even for Windows via WSL; and thus is a viable alternative way to run SILE.
+
+[Nix packages][nix] are available in both the stable and unstable channels.
+Because all packages are first made available in the unstable channel and then eventually make their way into the stable channel, to get the latest stable SILE releases we recommend hitting up the unstable channel first.
+You can use all the usual Nix tricks including launching a new shell with the `sile` command available or running it directly from any shell:
+
+```console
+$ nix shell nixos-unstable#sile
```

Comment:
Unfortunately there are a lot of mixups between the legacy `nix-shell` and the new easier to use `nix shell` that isn't quite stable yet. The current invocation should be:

```suggestion
$ nix shell nixpkgs/nixpkgs-unstable#sile
```